pkg/metrics/metrics.go

```diff
@@ -142,14 +142,34 @@ var (
 		[]string{"node_group"},
 	)
 	// NodeGroupScaleLock indicates if the nodegroup is locked from scaling
-	NodeGroupScaleLock = prometheus.NewSummaryVec(
-		prometheus.SummaryOpts{
+	NodeGroupScaleLock = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
 			Name: "node_group_scale_lock",
 			Namespace: NAMESPACE,
 			Help: "indicates if the nodegroup is locked from scaling",
 		},
 		[]string{"node_group"},
 	)
+	// NodeGroupScaleLockDuration indicates how long the nodegroup is locked from scaling
+	NodeGroupScaleLockDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "node_group_scale_lock_duration",
+			Namespace: NAMESPACE,
+			Help: "indicates how long the nodegroup is locked from scaling",
+			Buckets: []float64{1, 2, 3, 5, 8, 13, 21, 34, 55},
```

Comment:
Are these buckets in seconds? We might need to have larger buckets as the scale lock could be present for 5-10 minutes depending on how quickly the cloud provider can bring up the new instances and for them to join the cluster.

Reply:
I'll switch to a summary, given quantile results are wanted to be accurate (i.e. not interpolated via bucket values/frequency). Suggest the metric observations will be low enough to not put undue load on prometheus.